nidirect_workflow/src/Controller/AuditController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Drupal\nidirect_workflow\Controller;
+
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Controller\ControllerBase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class AuditController.
+ */
+class AuditController extends ControllerBase implements ContainerInjectionInterface {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * A logger instance.
+   *
+   * @var \Psr\Log\LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $account;
+
+  /**
+   * Creates a new ModerationStateConstraintValidator instance.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   * @param \Psr\Log\LoggerInterface $logger
+   *   The logger interface.
+   * @param \Drupal\Core\Session\AccountInterface $account
+   *   The current user.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerInterface $logger, AccountInterface $account) {
+    $this->entityTypeManager = $entity_type_manager;
+    $this->logger = $logger;
+    $this->account = $account;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity_type.manager'),
+      $container->get('logger.factory')->get('nidirect_workflow'),
+      $container->get('current_user')
+    );
+  }
+
+  /**
+   * Content Audit.
+   *
+   * @return string
+   *   Return confirmation string.
+   */
+  public function contentAudit($nid) {
+    $msg = 'Invalid node id';
```

Comment:
Wrap in `$this->t()` for translation. Does node id need to be node ID? Have a look in core and see what people have used there.

Reply:
Yes, good spot

Reply:
Also, add the $nid to the message.